errutil_test.go

```diff
@@ -149,6 +149,19 @@ errutil/test/dot.pkg dot.pkg.go:47 PkgFuncs
 errutil/test/dot.pkg dot.pkg.go:46 PkgFuncs
 	k1=v1`,
 		},
+		"WithStack": {
+			dotpkg.WithStack(),
+			errutil.Stack{
+				{Pkg: "errutil/test/dot.pkg", Func: "WithStack", File: "dot.pkg.go", Line: 55},
+				{Pkg: "errutil_test", Func: "TestBuildStack", File: "errutil_test.go", Line: 153},
+				{Pkg: "errutil_test", Func: "TestBuildStack", File: "errutil_test.go", Line: 153, Values: errutil.Tags{"msg": "regular"}},
```

Comment:
was attempting to have all the line info be exclusively in the dotpkg, but unavoidable for this case since it is the "real" stack, darn.